contracts/multisig-prover/src/query.rs

```diff
@@ -57,8 +57,10 @@ pub fn get_proof(
     })
 }
 
-pub fn current_verifier_set(deps: Deps) -> StdResult<Option<VerifierSet>> {
-    CURRENT_VERIFIER_SET.may_load(deps.storage)
+pub fn current_verifier_set(deps: Deps) -> StdResult<Option<(String, VerifierSet)>> {
+    CURRENT_VERIFIER_SET
+        .may_load(deps.storage)
+        .map(|op| op.map(|set| (set.id(), set)))
```

Comment:
Instead of keeping it separate and computing it new every time, shouldn't we instead just add the id as a field to the verifier set?